vuepress/docs/next/tutorials/README.md

```diff
@@ -7,18 +7,20 @@ Some screenshots or references to the previous version may be out-of-date and so
 only be available by building from source.
 :::
 
+
 # Learning Paths
 
-::: tip Entando simplifies the development of modern apps:
+::: tip Entando simplifies and automates the DevOps lifecycle of applications orchestrated by Kubernetes.
 
-1. Built using modern JavaScript frameworks
-2. Backed by microservices deployed in containers, and
-3. Orchestrated by Kubernetes for fully automated DevOps lifecycles
-:::
 
-Entando supports full stack micro frontend and microservice architectures for codebases that are easier to understand, maintain, and debug across large, distributed teams, and comes with a private component repository that makes it easy for teams to share and reuse components.
+* Build end-to-end modularity into your application for codebases that are easier to maintain and debug.
+* Share and reuse components from a common catalog between teams and applications.
+* Leverage the JHipster blueprint, standardizing and accelerating development.
+
+:::
+ 
 
-Use the navigation on the left to find step-by-step tutorials for common tasks or check out our learning paths below for a more structured approach.
+Check out the learning paths below, for varying levels of experience and needs for the developer.
```

Comment:
I think we discussed improving this - "varying levels of experience and needs" is awkward to me. Maybe "Check out the learning paths below to find step-by-step tutorials for the most common tasks you may encounter."